composer.json

```diff
@@ -70,7 +70,22 @@
         "release": "conventional-changelog --commit",
         "release:patch": "conventional-changelog --patch --commit",
         "release:minor": "conventional-changelog --minor --commit",
-        "release:major": "conventional-changelog --major --commit"
+        "release:major": "conventional-changelog --major --commit",
+        "git-hooks": "git config core.hooksPath git-hooks || exit 0",
+        "sass-prepare": "cd html/themes/custom/gms_ocha_common && npm install",
+        "sass-build": "cd html/themes/custom/gms_ocha_common && ./node_modules/.bin/sass sass:build",
+        "sass-cleanup": "cd html/themes/custom/gms_ocha_common && rm -rf ./node_modules",
```

Comment:
Theme is located at html/sites/all/themes/ocha_basic/

Reply:
I hope not, that's the Drupal 7 theme.

Reply:
There's no subtheme yet in this repo - from gms or from the d7 version.

Reply:
Yup, I just pasted in the sass script commands so they're there for when we do have a sub-theme. Nothing will try to actually run them at this point.